include/libtorrent/aux_/session_impl.hpp

```diff
@@ -902,7 +902,7 @@ namespace aux {
 #endif
 
 			// handles delayed alerts
-			mutable alert_manager m_alerts;
+			mutable aux::alert_manager m_alerts;
```

Comment:
these two changes shouldn't be necessary. This is already within the `aux` namespace.